minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-runtime/src/main/java/org/apache/nifi/minifi/MiNiFi.java

```diff
@@ -152,7 +152,9 @@ public void run() {
             }
 
             final long endTime = System.nanoTime();
-            logger.info("Controller initialization took " + (endTime - startTime) + " nanoseconds.");
+            final long durationNanos = endTime - startTime;
+            final long durationSeconds = TimeUnit.SECONDS.convert(durationNanos, TimeUnit.NANOSECONDS);
+            logger.info("Controller initialization took {} nanoseconds ({} seconds).", durationNanos, durationSeconds);
```

Comment:
This looks like it displays integer only. I think 1 significant digit of decimal precision would be useful (`0.6 seconds` vs. `0 seconds`). 

```
2018-07-23 13:29:10,774 INFO [main] org.apache.nifi.BootstrapListener Successfully initiated communication with Bootstrap
2018-07-23 13:29:10,774 INFO [main] org.apache.nifi.minifi.MiNiFi Controller initialization took 605462447 nanoseconds (0 seconds).
```